lbz/resource.py

```diff
@@ -101,24 +85,54 @@ def _get_user(self, headers: CIMultiDict) -> Union[None, User]:
         return None
 
     def _post_request_hook(self) -> None:
-        """
+        """*/
         Makes the post_request_hook run-time friendly.
         """
         try:
             self.post_request_hook()
         except Exception as err:  # pylint: disable=broad-except
             logger.exception(err)
 
+    @deprecated
```

Comment:
As long as these two methods are used automatically, I think everyone will get the deprecation warning no matter if they are using this code or not 🤔 

From my perspective removing them now is a better idea but I am open to discussion 🙂

Reply:
They will be removed in 0.6.0

Reply:
As we agreed, they stopped being used automatically, so it is fully ok to remove them later 👍